include/mbgl/annotation/annotation.hpp

```diff
@@ -20,26 +20,23 @@ class SymbolAnnotation {
     std::string icon;
 };
 
-using ShapeAnnotationGeometry = variant<
-    LineString<double>,
-    Polygon<double>,
-    MultiLineString<double>,
-    MultiPolygon<double>>;
+using ShapeAnnotationGeometry =
+    variant<LineString<double>, Polygon<double>, MultiLineString<double>, MultiPolygon<double>>;
 
 class LineAnnotation {
 public:
     ShapeAnnotationGeometry geometry;
-    style::PropertyValue<float> opacity { 1.0f };
```

Comment:
I'm a bit confused here - isn't `Cpp11BracedListStyle: false` supposed to preserve the whitespace here?